### ISSUE TITLE
Microsoft VBScript Regular Expressions 5.5 32 System32/vbscript.dll is ignored

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#125]
**Describe the bug**
Selecting Microsoft VBScript Regular Expressions 5.5 produces the error C:Windows\SysWOW64\vbscript.dll\3 unable to load type library reference.

The 32 bit VBScript in C:\Windows\System32 seems to be ignored.

**Comment by Kr00l** [comment]
SysWow64 is for 32 bit stuff and System32 for 64 bit stuff.

**Comment by FullValueRider** [comment]
Ah, is there no bounds to the depth of my ignorance.

The basic issue is that the implementation of the PredeclaredId atttribute allowed me to import the library supporting my AdventOfCode2020 attempts.  The library has over 2000 lines of actual code.  It imported fine (manual import) with the exception of 7 reported problems.  2 were related to the 'Like' operator which were easily resolved.  5 were related to the VBScript 5.5. regex.  

The WOW64 error is reported when I add the VBScript library

VBScript regex 5.5 appears in the list of registered com/activeX dll libraries

But the use of a regexp , as in

```
Private Type State

    Rx                  As VBScript_RegExp_55.RegExp
    ReplaceStr          As String
    
End Type
```
gives

```
unrecognized datatype symbol 'VBScript_RegExp_55'
```

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.9.2651.   Thanks for reporting!
